### PR TITLE
Minor typo fixes missed earlier

### PR DESCRIPTION
**Pull request by jasonbraganza**

### CODE REVIEW

**Comment by kushaldas** [comment]
This one has merge conflicts.

**Comment by jasonbraganza** [comment]
do I delete this and resubmit the changes?

File: docs/usage.rst [hunk 1]
```diff
@@ -90,15 +90,20 @@ Ask the next person to ask their question
 
     next
 
+<<<<<<< HEAD
 Stop taking quesions
 ====================
+=======
+Stop taking anymore questions
+============================
+>>>>>>> 2aa6a76... Minor typo fixes missed earlier
```
Comment (kushaldas) [inline docs/usage.rst]: I think this has many wrong lines :)

File: docs/usage.rst [hunk 2]
```diff
@@ -114,4 +119,8 @@ Can students ask questions now?
 
     #questions
 
+
 This gets reset to true, everytime we start a new session.
+=======
```
Comment (kushaldas) [inline docs/usage.rst]: Not sure about this one too.

**Review by kushaldas — CHANGES_REQUESTED** [review]

**Comment by jasonbraganza** [comment]
made changes and recommitted. Will that help?

**Pull request merged**